src/multiple-choice/components/app.tsx

```diff
@@ -30,10 +30,11 @@ export const App = () => {
     return () => window.removeEventListener("resize", resizeHandler);
   }, []);
 
+  const report = mode === "report";
```

Comment:
No need to change this now. But perhaps a good pattern for these components is to define 3 boolean variables:
```
runtimeMode, authoringMode, reportMode
```
And then logic like below will be consistent and concise.